### ISSUE TITLE
impaired/enhanced short-term object recognition memory

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3931]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. impaired short-term object recognition memory
2. enhanced short-term object recognition memory

>2. Synonyms (e.g., absent spleen)

1. short-term recognition memory deficit
1. decreased short-term recognition memory
2. increased short-term recognition memory

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. decrease in the ability of short-term memory to recognize objects during the first few minutes after training
2. increase in the ability of short-term memory to recognize objects during the first few minutes after training

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:38378836

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal short-term object recognition memory (MP:0002800)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
Added
impaired short-term object recognition memory - MP:0014365
enhanced short-term object recognition memory - MP:0014366

Also added the same for long-term
impaired long-term object recognition memory  - MP:0014367
enhanced long-term object recognition memory - MP:0014368

**Comment by sbello** [comment]
In the latest release
